pkg/pdatatest/go.mod

```diff
@@ -1,9 +1,10 @@
-module github.com/open-telemetry/opentelemetry-collector-contrib/internal/comparetest
+module github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest
```

Comment:
not sure if the module should be pdatatest or just pdata.

Reply:
I thought pdatatest is better here to not confuse with actual core/pdata module